packages/teams-js/src/private/remoteCamera.ts

```diff
@@ -58,6 +58,7 @@ export namespace remoteCamera {
    * @internal
    * Limited to Microsoft-internal use
    */
+  /* eslint-disable-next-line recommend-const-enums/recommend-const-enums */
```

Comment:
>eslint-disable-next-line recommend-const-enums/recommend-const-enums [](http://example.com/codeflow?start=5&length=68)

For all the places where this is now being suppressed, can we set a good example and also provide an explanation for why its being suppressed? Without a reason listed I worry that any future engineer will look at these existing instances and think that the standard is just to always suppress without being thoughtful about why